improver_tests/wxcode/wxcode/__init__.py

```diff
@@ -30,64 +30,598 @@
 # POSSIBILITY OF SUCH DAMAGE.
 """Utilities for Unit tests for Weather Symbols"""
 
-from improver.wxcode.utilities import get_parameter_names
+from typing import Any, Dict
 
 
-def check_diagnostic_lists_consistency(query):
-    """
-    Checks if specific input lists have same nested list
-    structure. e.g. ['item'] != [['item']]
+def prob_above_name(diagnostic: str) -> str:
+    """Inline function to construct probability cube name"""
+    return f"probability_of_{diagnostic}_above_threshold"
 
-    Args:
-        query (dict):
-            of weather-symbols decision-making information
 
-    Raises:
-        ValueError: if diagnostic query lists have different nested list
-            structure.
+LIGHTNING_VICINITY_PROB = prob_above_name(
+    "number_of_lightning_flashes_per_unit_area_in_vicinity"
+)
+CLOUD_NAME = "low_and_medium_type_cloud_area_fraction"
+CLOUD_PROB_ABOVE = prob_above_name(CLOUD_NAME)
+LOW_CLOUD_PROB_ABOVE = prob_above_name("low_type_cloud_area_fraction")
+TEXTURE_PROB_ABOVE = prob_above_name(f"texture_of_{CLOUD_NAME}")
+CONVECTION_PROB_ABOVE = prob_above_name("convective_ratio")
+PRECIP_PROB_ABOVE = prob_above_name("lwe_precipitation_rate")
+PRECIP_VICINITY_PROB_ABOVE = prob_above_name("lwe_precipitation_rate_in_vicinity")
+RAIN_PROB_ABOVE = prob_above_name("rainfall_rate")
+SLEET_PROB_ABOVE = prob_above_name("lwe_sleetfall_rate")
+SNOW_PROB_ABOVE = prob_above_name("lwe_snowfall_rate")
+VIS_PROB_BELOW = "probability_of_visibility_in_air_below_threshold"
 
-    """
-    diagnostic_keys = [
-        "diagnostic_fields",
-        "diagnostic_conditions",
-        "diagnostic_thresholds",
-    ]
-    values = [
-        get_parameter_names(query[key]) if key == "diagnostic_fields" else query[key]
-        for key in diagnostic_keys
-    ]
-    if not check_nested_list_consistency(values):
-        msg = "Inconsistent list structure: \n"
-        for key in diagnostic_keys:
-            msg += f"{key} = {query[key]}; \n"
-        raise ValueError(msg)
 
+def wxcode_decision_tree_uk() -> Dict[str, Dict[str, Any]]:
```

Comment:
This will be simplified to one example tree for testing as the unification work proceeds, but for now we need both here to preserve the existing unit tests.